src/plugin/regions.js

```diff
@@ -867,4 +883,25 @@ export default class RegionsPlugin {
 
         return min;
     }
+
+    /**
+     * Get the snapped to grid value.
+     *
+     * @param {*} value the value to snap to the grid, if needed
+     * @param {*} params the regions plugin params
+     */
```

Comment:
im sure there's a better type than `*` to document for these params.

Reply:
Like this?